src/systemd/cockpit-session@.service.in

```diff
@@ -7,3 +7,5 @@ StandardInput=socket
 StandardOutput=inherit
 StandardError=journal
 User=root
+# authentication failure or timeout, that's not a problem with the unit
+SuccessExitStatus=1 5 127
```

Comment:
I think I'd prefer to see `cockpit-session` exit with status 0 in these cases, unless this is just a transitory thing to get us across the transition...

Reply:
Exiting with 0 on an error leaves a bad taste in my mouth. For the time being we need to continue to support a direct ws → session execution. Several tests cover that, and it might also be relevant for more customized setups (Satellite/Foreman, reverse proxy, etc.) -- TBH I don't want to think about this right now, there's already a lot of complexity in this PR.

Reply:
Fair enough.